src/com/company/FootballMatch.java

```diff
@@ -19,21 +19,21 @@ public int getInt(){//валидатор ввода данных
         return num;
     }
 
-    //метод подсчёта исхода матча RealOutcome 1 и 2 - реальный счёт, Forecast 1 и 2 - прогноз пользователя
+   
     public int MatchForecast(int RealOutcome1,int RealOutcome2,int Forecast1, int Forecast2){
```

Comment:
better use mo understandable names of params, without numbers (RealOutcome1 = firstRealScore)